<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoView</class>
 <widget class="QWidget" name="InfoView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>586</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QTextBrowser" name="similarArtistInfo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <underline>false</underline>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextBrowser
{
background-color:transparent;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="placeholderText">
       <string>similar artists... </string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="tagsInfo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextBrowser
{
background-color:transparent;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="placeholderText">
       <string>releated tags...</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="artistText">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextBrowser
{
background-color:transparent;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QTextBrowser" name="albumText">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextBrowser
{
background-color:transparent;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="lyricsLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="customsearch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="4">
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="edit-find-replace">
            <normaloff>.</normaloff>.</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="titleLine">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Title...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="artistLine">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Artist...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QFrame" name="frame_6">
          <property name="maximumSize">
           <size>
            <width>1</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::VLine</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frame_3">
          <property name="maximumSize">
           <size>
            <width>1</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::VLine</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="lyricsText">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextBrowser
{
background-color:transparent;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
